### PR TITLE
Add label definitions list view

### PR DESCRIPTION
**Pull request by y-kkamil**
**Description**

Changes proposed in this pull request:

- Add label definitions list view 

**Related issue(s)**
See #1127

### CODE REVIEW

File: compass/src/components/MetadataDefinitions/MetadataDefinitions.component.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+      <b>{labelDef.key}</b>
+    </span>,
+    <span>{labelDef.schema ? 'true' : 'false'}</span>,
+  ];
+
+  render() {
+    const labelsDefinitionsQuery = this.props.labelDefinitions;
+    const labelsDefinitions = labelsDefinitionsQuery.labelDefinitions;
+
+    const loading = labelsDefinitionsQuery && labelsDefinitionsQuery.loading;
```
Comment (Wawrzyn321) [inline compass/src/components/MetadataDefinitions/MetadataDefinitions.component.js]: You don't need to check _labelsDefinitionsQuery_ for falsiness here, as PropTypes got it covered already.

File: compass/src/App.component.js [hunk 2]
```diff
@@ -63,6 +64,11 @@ class App extends React.Component {
             )}
           />
           <Route path="/scenarios" exact component={Scenarios} />
+          <Route
+            path="/metadatadefinitions"
```
Comment (Wawrzyn321) [inline compass/src/App.component.js]: Shouldn't we use hyphen-separated path names here, like in Console _config-maps_, _cmf-apis_ and so on?

Comment (y-kkamil) [reply compass/src/App.component.js]: you are correct, i missed it. fixed

File: compass/src/components/MetadataDefinitions/MetadataDefinitions.component.js (hunk 1 above)
Comment (y-kkamil) [reply compass/src/components/MetadataDefinitions/MetadataDefinitions.component.js]: great catch! fixed.

Comment (y-kkamil) [reply compass/src/components/MetadataDefinitions/MetadataDefinitions.component.js]: great catch! fixed.

**Comment by y-kkamil** [comment]
> Search crash everything
> Screen Shot 2019-08-08 at 13 59 17

you are right, it is fixed in #1148

File: compass/src/components/MetadataDefinitions/MetadataDefinitions.component.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+import PropTypes from 'prop-types';
+
+import GenericList from '../../shared/components/GenericList/GenericList';
+import CreateLabelModal from '../Labels/CreateLabelModal/CreateLabelModal.container';
+
+class MetadataDefinitions extends React.Component {
+  headerRenderer = () => ['Labels', 'Schema Provided'];
+
+  rowRenderer = labelDef => [
+    <span className="link">
```
[the remark this answers is not in the document]
Comment (y-kkamil) [reply compass/src/components/MetadataDefinitions/MetadataDefinitions.component.js]: i think, the details view is on it's way in #1151

**Pull request merged**